### DISCUSSION TITLE
Migration from Trillium to other note taking software

### DISCUSSION THREAD

**Discussion by jjolatunji1** [TriliumNext/Trilium#2772] [General]
Hello, Can someone help me with migrating from Trilium to Obsidian?

Thanks in advance.

**Comment by Rayme** [comment]
You can try to right click on the root tree then select export, choose markdown, it will export all your notes to a zip file. Extract it and open it in obsidian as a vault.
There is an issue for the export to markdown function. All heading 2 will turn into **bold** text, lists will be `*` instead of `-`
